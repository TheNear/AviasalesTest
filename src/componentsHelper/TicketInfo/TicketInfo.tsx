import React from "react";

function TicketInfo(): JSX.Element {
  return (
    <div>
      <div />
    </div>
  );
}

export default TicketInfo;
