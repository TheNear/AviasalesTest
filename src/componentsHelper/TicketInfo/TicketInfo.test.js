import React from "react";
import { shallow } from "enzyme";
import TicketInfo from "./TicketInfo";

describe("Test custom TicketInfo component", () => {
  it("Render empty TicketInfo", () => {
    const wrapper = shallow(<TicketInfo />);
    expect(wrapper).toMatchSnapshot();
  });
});
