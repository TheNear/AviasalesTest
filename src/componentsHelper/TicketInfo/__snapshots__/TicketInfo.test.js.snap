// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test custom TicketInfo component Render empty TicketInfo 1`] = `
<div
  className="container"
>
  <div
    className="title"
  />
  <div
    className="text"
  >
    - / -
  </div>
</div>
`;
